addons/project/models/project_project.py

```diff
@@ -21,6 +21,9 @@ class Project(models.Model):
     _rating_satisfaction_days = 30  # takes 30 days by default
     _check_company_auto = True
 
+    def _get_hide_partner(self):
+        return False
+
```

Comment:
Can you move this method here instead? 
https://github.com/odoo/odoo/blob/d84f59d6564ddaae5e6ee120a5b25a83445efac0/addons/project/models/project_project.py#L772

Thanks in advance! 🙂